test/RouteResultTest.php

```diff
@@ -68,24 +48,12 @@ public function testRouteFailureRetrieveHttpMethods()
         $this->assertSame([], $result->getAllowedMethods());
     }
 
-    public function testRouteRetrieveHttpMethods()
```

Comment:
[Yes](https://github.com/michaelmoussa/zend-expressive-router/blob/1c7411f626919cffb901e22e34de76ead598e61a/test/RouteResultTest.php#L90-L102) (copy-pasta same link as other comment for completeness)